scripts/typescript-compatibility-build.yml

```diff
@@ -1,10 +1,8 @@
 trigger:
   branches:
     include:
-      - tsc-compat-debug
-  tags:
-    include:
-      - office-ui-fabric-react_v*
+      - master
```

Comment:
```suggestion

```
@aftab-hassan, I'm worried this will build too often given the frequency code is committed to `master`. Can we stick with tags and scheduled builds for now?

Reply:
@KevinTCoughlin ,thanks for the suggestion, makes sense.